data-prepper-plugins/kafka-plugins/src/test/resources/kafka-pipeline-auth-insecure.yaml

```diff
@@ -5,7 +5,7 @@ log-pipeline :
           - "localhost:9092"
         encryption:
           type: "NONE"
-          certificate_content: "CERTIFICATE_DATA"
+          certificate: "CERTIFICATE_DATA"
```

Comment:
Shouldn't this fail? What could we put here that wouldn't start to fail if validated up-front?

Reply:
This is only used in Unit test and has not been through Jakarta yet.